### DISCUSSION TITLE
Please provide the package file for latest version (22.1.3)

### DISCUSSION THREAD

**Discussion by Mouri-Kogoro** [ArcadeData/arcadedb#440] [General]
After version 22.1.3 the release page doesn't have the package files of the subsequent versions. 

[https://github.com/ArcadeData/arcadedb/releases](https://github.com/ArcadeData/arcadedb/releases)

The problem is that I don't know how to use the source files. So please provide the package file for latest version (or provide instruction how to use that source files).

**Comment by kerosumi** [comment]
Hi Arcadedb team,
I am a newbie and found your project very interesting. But I found your documentation page not very beginner friendly.May be you are looking for only experienced programmers. 
For example running server from bin/server.sh is not correct as there is no bin folder in your source code files.
I know you are still starting up this project but please ,please provide proper detailed documentation on how to start.

**Reply by lvca** [reply]
@kerosumi the documentation refers to the binary version of ArcadeDB. You should find the `bin` directory in the package. If you're compiling from the sources, you can get your binaries by executing:

`mvn clean install -DskipTests`

And then find your binaries under package/target.

**Reply by kerosumi** [reply]
> @kerosumi the documentation refers to the binary version of ArcadeDB. You should find the `bin` directory in the package. If you're compiling from the sources, you can get your binaries by executing:
> 
> `mvn clean install -DskipTests`
> 
> And then find your binaries under package/target.

Thanks Luca ,I figured it out.Also git cloned the repository as you suggested elsewhere. I hope to see this going forward.If it does I hope it will be a great db system. But please spend some time on a great manual. Still few things I could not figure out.
/restore.sh ,graphml import through studio etc etc.

**Comment by lvca** [comment]
@Mouri-Kogoro you're right, just uploaded the binaries for 22.6.1: https://github.com/ArcadeData/arcadedb/releases/tag/22.6.1.

**Reply by gudzpoz** [reply]
Is there a plan to publish newer versions to Maven Central? The latest version there is still 22.2.1, and the Github action seems to be failing.

**Reply by lvca** [reply]
@arcade-player is working on restoring the automatic build from our CI.
